sources/packages/web/src/views/aest/SearchInstitutions.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <full-page-container>
+    <h2 class="color-blue">Search Institution</h2>
+    <v-row class="mt-5">
+      <v-col><label for="legalName">Legal Name</label></v-col>
+      <v-col><label for="operatingName">Operating Name</label></v-col>
+      <v-col></v-col>
```

Comment:
why empty 2 `<v-col></v-col>`. same in line 8, 13 ?

Reply:
changed to a different input and removed empty cols